mysql-test/suite/rocksdb/t/use_direct_reads_writes.test

```diff
@@ -5,43 +5,37 @@
 # caused an assertion in RocksDB.  Now it should not be allowed and the
 # server will not start with that configuration
 
-# Write file to make mysql-test-run.pl expect the "crash", but don't restart
-# the server until it is told to
---let $_server_id= `SELECT @@server_id`
---let $_expect_file_name= $MYSQLTEST_VARDIR/tmp/mysqld.$_server_id.expect
---exec echo "wait" >$_expect_file_name
-shutdown_server 10;
+--let SEARCH_FILE=$MYSQLTEST_VARDIR/tmp/rocksdb.use_direct_reads_writes.err
 
-# Clear the log
---exec echo "" >$MYSQLTEST_VARDIR/log/mysqld.1.err
+# Expect server crash on startup with invalid combination of arguments
+--source include/expect_crash.inc
+shutdown_server 10;
```

Comment:
And replace shutdown_server with shutdown_mysqld.inc. That it will write a non-crash-expecting MTR file is immaterial, it will be overwritten by expect_crash.inc, OTOH the "10" timeout is very problematic with Valgrind runs etc.